### PR TITLE
PR #3681 followup

### PR DESCRIPTION
**Pull request by Sija**
PR #3681 followup which fixes few missing instances of `Slice(UInt8)`.

### CODE REVIEW

File: src/base64.cr [hunk 1]
```diff
@@ -153,7 +153,7 @@ module Base64
     strict_encode_to_io_internal(data, io, CHARS_SAFE, pad: true)
   end
 
-  # Returns the Base64-decoded version of `data` as a *Bytes*.
+  # Returns the Base64-decoded version of `data` as a `Bytes`.
```
Comment (RX14) [inline src/base64.cr]:
```cr
# *data* as a `Bytes`
```

Comment (Sija) [inline src/base64.cr]: I've changed it earlier but there are multiple instances of such notation so I could replace throughout the file if needed.

Comment (RX14) [inline src/base64.cr]: https://crystal-lang.org/docs/conventions/documenting_code.html says to use `*` for param names and `\`` for types, but if it's wrong throughout the file that should probably be a separate PR.

Comment (Sija) [inline src/base64.cr]: @RX14 True that, I can send separate PR with changes.

**Comment by asterite** [comment]
Thanks!
